XamlControlsGallery/ControlExample.xaml

```diff
@@ -80,23 +95,41 @@
                     <RowDefinition Height="Auto" />
                 </Grid.RowDefinitions>
                 <Grid.ColumnDefinitions>
+                    <ColumnDefinition />
```

Comment:
Should this be Width="*"?

It seems like something in this change is causing the samples to be horizontally squished. 

Before:
![image](https://user-images.githubusercontent.com/25991996/90069539-498f5200-dca7-11ea-8e85-14323e96c65d.png)


After:
![image](https://user-images.githubusercontent.com/25991996/90069577-59a73180-dca7-11ea-9a86-76dbba780939.png)

Reply:
Just kidding - I actually think the next column needs to have Width = Auto.

What is happening right now is that neither the sample or the output column have their width set, so I think they both default to *, when the output column should actually collapse. 

That said, I don't know if that makes the screenshots too wide for some samples.